docs/Installing.asciidoc

```diff
@@ -633,6 +633,17 @@ The openQA web UI should be available on http://localhost/ now. To simply
 start openQA without enabling it permanently one can simply use `systemctl
 start` instead.
 
+== Enable automatic database backup
+An optional systemd service, `openqa-dump-db.service`, can be enabled to
+perform daily database backups. This service is triggered by the
+`openqa-dump-db.timer`. To enable automatic database backup, run:
+
+[source,sh]
+--------------------------------------------------------------------------------
+systemctl enable --now openqa-dump-db.timer
+--------------------------------------------------------------------------------
+Backups are stored at `/var/lib/openqa/backup`.
+
```

Comment:
Please move this section further down, e.g. into "Advanced configuration". This section focusses on getting new instances up and running in the "basic configuration" and an optional database backup doesn't fit here.